receiver/redisreceiver/config.go

```diff
@@ -0,0 +1,8 @@
+package redisreceiver
+
+import "github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+
+type Config struct {
+	configmodels.ReceiverSettings `mapstructure:",squash"`
```

Comment:
Please add a README with configuration information -- see other receivers for an example

Reply:
Sounds good. Will do.